tests/tests/src/lampo_cln_tests.rs

```diff
@@ -539,3 +540,101 @@ fn be_able_to_kesend_payments() {
     assert!(result.is_ok(), "{:?}", result);
     async_run!(cln.stop()).unwrap();
 }
+
+#[test]
+fn close_channel() {
+    init();
+    let cln = async_run!(cln::Node::with_params(
+        "--developer --dev-bitcoind-poll=1 --dev-fast-gossip --dev-allow-localhost",
+        "regtest"
+    ))
+    .unwrap();
+    std::thread::sleep(Duration::from_secs(2));
+    let btc = cln.btc();
+    let lampo_manager = LampoTesting::new(btc.clone()).unwrap();
+    let lampo = lampo_manager.lampod();
+    let _info: response::GetInfo = lampo.call("getinfo", json::json!({})).unwrap();
+    let info_cln = cln.rpc().getinfo().unwrap();
+    let events = lampo.events();
+    let address = lampo_manager.fund_wallet(101).unwrap();
+    wait!(|| {
+        let Ok(Event::OnChain(OnChainEvent::NewBestBlock((_, height)))) =
+            events.recv_timeout(Duration::from_millis(100))
+        else {
+            return Err(());
+        };
+        if height.to_consensus_u32() == 101 {
+            return Ok(());
+        }
+        Err(())
+    });
+    let _: json::Value = lampo
+        .call(
+            "fundchannel",
+            request::OpenChannel {
+                node_id: cln.rpc().getinfo().unwrap().id,
+                port: Some(cln.port.into()),
+                amount: 1_500_000_000,
+                public: true,
+                addr: Some("127.0.0.1".to_owned()),
+            },
+        )
+        .unwrap();
+
+    std::thread::sleep(Duration::from_secs(2));
+
+    // Get the transaction confirmed
+    let _ = btc.rpc().generate_to_address(6, &address).unwrap();
+    wait!(|| {
+        log::info!(target: "tests", "wait for confimetion");
+        let _ = btc.rpc().generate_to_address(1, &address).unwrap();
+        // Get the transaction confirmed
+        for _ in 0..100 {
+            let Ok(event) = events.recv_timeout(Duration::from_nanos(100)) else {
+                continue;
+            };
+            log::info!(target: "tests", "lampo event: {:?}", event);
+            match event {
+                Event::Lightning(LightningEvent::ChannelReady { .. }) => return Ok(()),
+                _ => continue,
+            };
+        }
+        Err(())
+    });
+
+    wait!(|| {
+        let channels = cln.rpc().listfunds().unwrap().channels;
+        if channels.is_empty() {
+            return Err(());
```

Comment:
I updated the tests I removed this style of writing, could you update also this version of the code with the one that I am using right now?

e.g: https://github.com/vincenzopalazzo/lampo.rs/blob/main/tests/tests/src/lampo_cln_tests.rs#L342

Reply:
I don't get it. Aren't they both the same?

Reply:
So why we need a `std::thread::sleep(Duration::from_secs(2));`? I do not undestnad

Reply:
Oh Okay, removed it thanks.